aspnetcore/host-and-deploy/windows-service.md

```diff
@@ -220,6 +220,39 @@ If the custom `WebHostService` code requires a service from dependency injection
 
 Services that interact with requests from the Internet or a corporate network and are behind a proxy or load balancer might require additional configuration. For more information, see <xref:host-and-deploy/proxy-load-balancer>.
 
+## Configure HTTPS
+
+A Windows Service runs under the [LocalSystem](/windows/desktop/services/localsystem-account) account. Specify an HTTPS port and [X509Certificate2](/dotnet/api/system.security.cryptography.x509certificates.x509certificate2) in a [Kestrel server endpoint configuration](xref:fundamentals/servers/kestrel#endpoint-configuration). 
```

Comment:
Organization? The current order of this new content seems like a series of non-sequiturs.

Reply:
I didn't write this one from scratch ... I inherited it and have only made updates to it since then. I agree with your assessment.

The current outline is ...

* Host ASP.NET Core in a Windows Service
  - Get started
  - Provide a way to run outside of a service
  - Handle stopping and starting events
  - Proxy server and load balancer scenarios
  - Configure HTTPS
  - Additional resources

I opened https://github.com/aspnet/Docs/issues/7898 for the work. I'd do it now, except that I'm 🏃🏃🏃🏃😅 this sprint and the next one (possibly the next two sprints). Of course, the purpose here is just to surface the HTTPS content. The best spot given the current outline is probably to place it at the end.

Reply:
I was referring to the new sentences and their internal order.

Reply:
Yes ... the first sentence is out of place. I'll fix it on the next commit.

Reply:
Great :eye: on that one.

... but I still think I'd like to work on the section headings, so I'm going to leave #7898 open.